_pages/en/strategy-product-management.md

```diff
@@ -0,0 +1,475 @@
+| ------------------------| ------------------------------------------------------------------------ |
+| Privacy                 | The Business ADM takes on the liability of a privacy incident for her product.<br>Assurances involves performing a [Privacy Impact Assessment](https://www.priv.gc.ca/en/privacy-topics/privacy-impact-assessments/gd_exp_202003/#toc4) and interacting with the [Privacy Management Division](https://esdc.prv/en/csb/privacy-management.shtml)|
+| Cyber Security          | The Business ADM takes on the liability of a cyber security incident for her product.<br>Assurances involves applying [ITSG-33 controls](https://cyber.gc.ca/en/guidance/overview-itsg-33) on the product proportional to the Statement of Sensitivity and Statement of Acceptable Risks exercises.|
+| Ethics                  | The Business ADM takes on the liability of an ethics incident for her product.<br>Assurances involves producing an [Algorithmic Impact Assessment](https://www.canada.ca/en/government/system/digital-government/digital-government-innovations/responsible-use-ai/algorithmic-impact-assessment.html) and meeting the [Directive on Automated Decision Making](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32592) requirements.|
+| Production Release      | The Business ADM takes on the liability of releasing her product in production by following ESDC’s [Software Release Process](https://014gc.sharepoint.com/:b:/r/sites/SDLCActionPlanTeam/Shared%20Documents/General/Software%20Development%20and%20Release%20Governance.pdf?csf=1&web=1&e=SdnmEy) (as per ITSM).<br>The CIO may establish a “capped size” for releases (to protect limited network bandwidth on premise and greater ESDC reputation) |
+| Corporate Services     | The Business ADM funds its own IT corporate services to function, including production and incident management operations |
+
+_Note 1:_ A delegation of all areas would indicate more than one CIO and DOCS reporting to the Deputy Head.
+Highly unlikely.
+
+_Note 2:_ A delegation in one area grants a Business ADM the ability to embed personnel of that area under its HR Org structure.
+
+_Note 3:_ Production Release process maps to the [IITB SDLC](https://014gc.sharepoint.com/:b:/r/sites/SDLCActionPlanTeam/Shared%20Documents/General/Software%20Development%20and%20Release%20Governance.pdf?csf=1&web=1&e=SdnmEy).
+Quality Assurances that the right stakeholders were involved throughout the s/w lifecycle before releasing to production.
+
+_Note 4:_ Corporate services include Funding for Cyber Security Operation Centre (CSOC), IT Security Assessment, Cloud Operations, Architecture Review, Service Desk, Laptops, VPNs.
+
+## Appendix C - Product Framework Overview
+
+<div style="display: block; width: inherit; text-align: center; line-height: 3;">
+    <a href="/ITStrategy/assets/images/product-management-framework-overview.png" style="display: block">
+        <img src = "{{ site.baseurl }}/assets/images/product-management-framework-overview.png"
```

Comment:
I think at Step 5, 1st bullet, "with" should be removed, like this:
"Provides Funding Source to..."

Also very minor, but Step 4, 3rd bullet, should have a "," after "e.g." like this: "e.g.,"